Formula/vgmstream.rb

```diff
@@ -35,11 +49,15 @@ class Vgmstream < Formula
   fails_with gcc: "5" # ffmpeg is compiled with GCC
 
   def install
-    system "cmake", "-DBUILD_AUDACIOUS:BOOL=OFF", *std_cmake_args, "."
-    system "cmake", "--build", ".", "--config", "Release", "--target", "vgmstream_cli", "vgmstream123"
-    bin.install "cli/vgmstream-cli"
-    bin.install "cli/vgmstream123"
-    lib.install "src/libvgmstream.a"
+    ENV["LIBRARY_PATH"] = "#{HOMEBREW_PREFIX}/lib"
+    Dir.mkdir("build")
+    Dir.chdir("build") do
+      system "cmake", "-DBUILD_AUDACIOUS:BOOL=OFF", ".."
+      system "make"
+      bin.install "cli/vgmstream-cli"
+      bin.install "cli/vgmstream123"
+      lib.install "src/libvgmstream.a"
+    end
```

Comment:
Do we need to change this?

Reply:
vgmstream's [build document](https://github.com/vgmstream/vgmstream/blob/master/doc/BUILD.md#macos) suggests these command and [doesn't recommend build in root folder](https://github.com/vgmstream/vgmstream/blob/master/doc/BUILD.md#cmake-builds).

Reply:
Ok, let's do this instead then
```suggestion
    system "cmake", "-S", ".", "-B", "build", "-DBUILD_AUDACIOUS:BOOL=OFF", *std_cmake_args
    system "cmake", "--build", "build"
    bin.install "build/cli/vgmstream-cli", "build/cli/vgmstream123"
    lib.install "build/src/libvgmstream.a"
```